components/authz-service/storage/v2/memstore/memstore.go

```diff
@@ -243,6 +243,7 @@ func (s *State) GetPolicyChangeNotifier(ctx context.Context) (v2.PolicyChangeNot
 }
```

Comment:
ℹ️ Memstore adaptations. I anticipate that we all kind of have some of these in our current WIP PRs... :/

Reply:
If we wanna match it as close to postgres as possible, I think we'd want a staged cache and an applied cache. Is that worthwhile? Probably not.

Reply:
Heh probably not as much WIP as we should. Left a comment in Slack.